packages/synthetix-main/contracts/storage/CollateralConfiguration.sol

```diff
@@ -74,13 +77,14 @@ library CollateralConfiguration {
     }
 
     function getCollateralPrice(Data storage self) internal view returns (uint) {
-        (, int256 answer, , , ) = IAggregatorV3Interface(self.priceFeed).latestRoundData();
+        OracleManager.Data memory oracleManager = OracleManager.load();
+        Node.Data memory node = IOracleManagerModule(oracleManager.oracleManagerAddress).process(self.oracleNodeId);
 
         // sanity check
         // TODO: this will be removed when we get the oracle manager
-        require(answer > 0, "The collateral value is 0");
+        require(node.price > 0, "The collateral value is 0");
```

Comment:
we may be able to remove this require now that we have the oracle manager, it can verify this instead